pylint/testutils/output_line.py

```diff
@@ -12,30 +11,15 @@
 from pylint.testutils.constants import UPDATE_OPTION
 
 
-class MessageTest(
-    collections.namedtuple(
-        "MessageTest", ["msg_id", "line", "node", "args", "confidence"]
-    )
-):
+class MessageTest(NamedTuple):
```

Comment:
🙏 My goodness
![feelgoodman](https://user-images.githubusercontent.com/5493666/142939383-8f7a3adc-1ccc-4972-9e9a-926beef4be49.png)

But is this already possible in python 3.6 ?

Reply:
Yes! Or at least it should. The default value only from `3.6.1` but we are already depending on that in other PRs for `2.12` as well.

Reply:
If I remember correctly, the change was planned for `2.12.1`. Would have to look up why though.

Reply:
I think because of the default value of `NamedTuple`. However, the `end_line` PR also used those.

Reply:
True. With the exception in place, someone with `3.6.0` won't even get that far. `pip install pylint` will fail. It's probably an idea to pick up most of #5068 then too, except for bumping the `python_requires` version and removing the exception. 

https://github.com/PyCQA/pylint/blob/9e32192fe7bf77281d0dfbc107984b751983e113/setup.py#L17-L18